image/cli/mascli/functions/pipeline_config_additional_configs

```diff
@@ -28,7 +37,9 @@ function config_pipeline_additional_configs() {
     echo
     prompt_for_confirm "Are these the correct configuration files to apply?" || exit 0
 
-    $DIR/tools/cfgmgr add $CONFIG_DIR/additional-configs-$MAS_INSTANCE_ID.yaml $LOCAL_MAS_CONFIG_DIR
+    cp $DIR/templates/additional-configs.yaml $CONFIG_DIR/additional-configs-$MAS_INSTANCE_ID.yaml
+    python3 $DIR/tools/cfgmgr add $CONFIG_DIR/additional-configs-$MAS_INSTANCE_ID.yaml $LOCAL_MAS_CONFIG_DIR
+    CONFIGURE_EXTERNAL_DB="true"
```

Comment:
Why was this added?  It's serving no value, we already have `db2_action_manage` that tells us whether or not we are setting up a database for manage, this appears to be completely redundant to me.

Reply:
Fixed it myself